spec/FrameSpec.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
```

Comment:
Really clearly laid out tests. Easy to understand what you're testing. The only thing I might suggest is subdividing the tests further using nested describes.